Source/WTF/wtf/Vector.h

```diff
@@ -1986,6 +1987,8 @@ inline Vector<typename CopyOrMoveToVectorResult<Collection>::Type> moveToVector(
     return moveToVectorOf<typename CopyOrMoveToVectorResult<Collection>::Type>(collection);
 }
 
+template<typename U> Vector(const U*, size_t) -> Vector<U>;
```

Comment:
Maybe elaborate what change you're making to Vector in the commit message?

Reply:
Good point. Will update.